### PR TITLE
Remove calorie command

### PR DESCRIPTION
**Pull request by wayne987**
remove a certain calorie from a particular day
example: remove tp/out d/2020-10-23 i/2
removes a calorie output from day:2020-10-23 with the index of 2

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RemoveCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.calorie.Calorie;
+import seedu.address.model.calorie.CalorieManager;
+import seedu.address.model.day.Day;
+
+
```
Comment (junhui-phoon) [inline src/main/java/seedu/address/logic/commands/RemoveCommand.java:20]: Javadocs header

File: src/main/java/seedu/address/logic/parser/CalorieCommandParser.java [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
-import seedu.address.model.day.calorie.Input;
-import seedu.address.model.day.calorie.Output;
-import seedu.address.model.day.calorie.Time;
+import seedu.address.model.calorie.Calorie;
+import seedu.address.model.calorie.CalorieCount;
+import seedu.address.model.calorie.Exercise;
+import seedu.address.model.calorie.Food;
+import seedu.address.model.calorie.Input;
+import seedu.address.model.calorie.Output;
+import seedu.address.model.calorie.Time;
```
Comment (junhui-phoon) [inline src/main/java/seedu/address/logic/parser/CalorieCommandParser.java:24]: Javadocs header

File: src/main/java/seedu/address/logic/parser/RemoveCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+
+import java.time.LocalDate;
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.RemoveCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
```
Comment (junhui-phoon) [inline src/main/java/seedu/address/logic/parser/RemoveCommandParser.java:15]: Javadocs header

File: src/main/java/seedu/address/model/calorie/CalorieManager.java [hunk 4]
```diff
@@ -1,10 +1,11 @@
-package seedu.address.model.day.calorie;
+package seedu.address.model.calorie;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.logging.Logger;
 
 import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.core.index.Index;
```
Comment (junhui-phoon) [inline src/main/java/seedu/address/model/calorie/CalorieManager.java:9]: Javadocs header

**Review by junhui-phoon — CHANGES_REQUESTED** [review]
Add missing Javadocs header to the classes under model/calorie package

File: src/main/java/seedu/address/logic/parser/RemoveCommandParser.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RemoveCommand.MESSAGE_USAGE), pe);
+        }
+
+        String type = ParserUtil.parseType(argMultimap.getValue(PREFIX_CALORIE_TYPE).get());
+
+        LocalDate date = ParserUtil.parseLocalDate(argMultimap.getValue(PREFIX_DATE).get());
+        return new RemoveCommand(index, type, date);
+    }
+
+    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
Comment (ethanso56) [inline src/main/java/seedu/address/logic/parser/RemoveCommandParser.java:46]: add javadocs

Comment (ethanso56) [reply src/main/java/seedu/address/logic/parser/RemoveCommandParser.java:51]: for the addPrefixesPresent

**Review by ethanso56 — COMMENTED** [review]
looks good to me

**Pull request merged**